spec/grape/validations_spec.rb

```diff
@@ -352,6 +352,49 @@ def define_requires_none
       end
     end
 
+    # Ensure there is no leakage of indices between requests
+    context 'required with a hash inside an array' do
+      before do
+        subject.params do
```

Comment:
Is `subject` being cleared between tests or are we calling `params` on the same `Grape::API`?